integration/plugin/graphdriver/external_test.go

```diff
@@ -395,12 +395,12 @@ func testGraphDriverPull(c client.APIClient, d *daemon.Daemon) func(*testing.T)
 		defer d.Stop(t)
 		ctx := context.Background()
 
-		r, err := c.ImagePull(ctx, "busybox:latest@sha256:bbc3a03235220b170ba48a157dd097dd1379299370e1ed99ce976df0355d24f0", types.ImagePullOptions{})
+		r, err := c.ImagePull(ctx, "busybox:latest@sha256:d366a4665ab44f0648d7a00ae3fae139d55e32f9712c67accd604bb55df9d05a", types.ImagePullOptions{})
```

Comment:
I see that originally, I made this digest match the one that we used in the "frozen images"; https://github.com/moby/moby/pull/37477, but we forgot to update this test to match in the last update; https://github.com/moby/moby/pull/41145

Could you instead update to `95cf004f559831017cdf4628aaf1bb30133677be8702a8c5f2994629f637a209` (what's currently used in the frozen images)?

Alternatively, update the frozen images to match this digest, but I don't think we need to update those (unless really needed). Guess it would be good to add a comment there though, so that we don't forget to update in future

Reply:
thanks! I have updated the digest to `95cf004f559831017cdf4628aaf1bb30133677be8702a8c5f2994629f637a209`, please check.